docs/contributors/testing-overview.md

```diff
@@ -387,14 +387,26 @@ or interactively
 npm run test-e2e:watch
 ```
 
-Sometimes it's useful to observe the browser while running tests. To do so you can use these environment variables:
+Sometimes it's useful to observe the browser while running tests. Then, use this command:
+
+```bash
+npm run test-e2e:open-browser
+```
+
+You can control the speed of execution by changing environment variables like below:
 
 ```bash
 PUPPETEER_HEADLESS=false PUPPETEER_SLOWMO=80 npm run test-e2e:watch
```

Comment:
There is an option that does the same:
```suggestion
npm run test-e2e:watch -- --puppeteer-interactive --puppeteer-slowmo=200
```

https://github.com/WordPress/gutenberg/blob/33a821ff00755e7a535e30b69bb0588941c3b908/packages/scripts/scripts/test-e2e.js#L59-L62